src/Components/ArtworkGrid/ArtworkGrid.tsx

```diff
@@ -117,7 +117,7 @@ export class ArtworkGridContainer extends React.Component<
             artwork={artwork}
             key={artwork.id}
             mediator={this.props.mediator}
-            lazyLoad={i + j >= preloadImageCount}
+            lazyLoad={i * columnCount + j >= preloadImageCount}
```

Comment:
Since i,j both start at 0, plus there's a `>=`, so you're doing a bit of math, and as such seems ripe for a subtle 'off-by-1' error or what have you (not unlike the current issue).

It might be clearer to have an `imageCount` that you instantiate to 0 outside the loops and increment inside the inner loop. Then the prop is `lazyLoad={imageCount > preloadImageCount}`

This might remove any math altogether!

This is an overly pedantic suggestion btw, nice find.

Reply:
I second what @mzikherman says above. This does currently look like code I could easily mess up; a named variable would help me better understand it. 

It also might be helpful to introduce test coverage of this counting, somehow.

Reply:
I started going down this road, but as the loops are now this wouldn't work. When it's laying  out the grid, it lays out the columns first. So it'd fill up the left-most column then it'd fill up the right-most column. We could add it to a variable if we flipped the loops. 

@pepopowitz yeah, I'll look into adding a test too.